packages/core/EventManager.ts

```diff
@@ -97,7 +97,7 @@ export default class EventManager {
     // (type closecom_other_calendar)
     this.calendarCredentials = appCredentials.filter((cred) => cred.type.endsWith("_calendar"));
     this.videoCredentials = appCredentials
-      .filter((cred) => cred.type.endsWith("_video"))
+      .filter((cred) => cred.type.endsWith("_video") || cred.type.endsWith("_conferencing"))
```

Comment:
I am not sure why it's needed @joeauyeung as we are still using _video postfix for conferencing apps and using the same for shimmer app too . Leaving it as is for now.